### PR TITLE
Adfs Module: Improve Inputs and Outputs for the AdfsClient Cmdlets

### PR DESCRIPTION
**Pull request by X-Guardian**
This PR improves the descriptions for the Inputs and Outputs section of the following `AdfsClient` cmdlets:

- Add-AdfsClient
- Disable-AdfsClient
- Enable-AdfsClient
- Get-AdfsClient
- Remove-AdfsClient
- Set-AdfsClient

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/adfs/Add-AdfsClient.md [hunk 1]
```diff
@@ -317,11 +317,15 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+### System.String
+
+String objects are received by the *ClientId*, *Description*, *Name*, and *RedirectUri* parameters.
 
 ## OUTPUTS
 
-### System.Object
+### Microsoft.IdentityServer.Management.Resources.AdfsClient
+
+Returns the new AdfsClient object when the PassThru parameter is specified. By default, this cmdlet does not generate any output.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Add-AdfsClient.md]:
```suggestion
Returns the new AdfsClient object when the _PassThru_ parameter is specified. By default, this cmdlet does not generate any output.
```

Comment (JohanFreelancer9) [inline docset/windows/adfs/Add-AdfsClient.md]:
```suggestion
Returns the new AdfsClient object when the *PassThru* parameter is specified. By default, this cmdlet does not generate any output.
```

Comment (JohanFreelancer9) [inline docset/windows/adfs/Add-AdfsClient.md]:
```suggestion
Returns the new AdfsClient object when the _PassThru_ parameter is specified. By default, this cmdlet does not generate any output.
```

File: docset/windows/adfs/Disable-AdfsClient.md [hunk 2]
```diff
@@ -153,8 +153,20 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+
+### System.String
+
+String objects are received by the *TargetClientId* and *TargetName* parameters.
+
 ## OUTPUTS
 
+### Microsoft.IdentityServer.Management.Resources.AdfsClient
+
+Returns the disabled AdfsClient object when the PassThru parameter is specified. By default, this cmdlet does not generate any output.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Disable-AdfsClient.md]:
```suggestion
Returns the disabled AdfsClient object when the **_PassThru_** parameter is specified. By default, this cmdlet does not generate any output.
```

**Comment by e0i** [comment]
@get-itips Please let us know of any additional revision suggestions. Thanks.

**Pull request merged**